client/securedrop_client/sdk/README.md

```diff
@@ -0,0 +1,10 @@
+# SecureDrop SDK
+
+This SDK provides a Pythonic interface to the [SecureDrop Journalist Interface
+API](https://developers.securedrop.org/en/latest/journalist_api.html).
+
+* It SHOULD provide types and methods for all resources and endpoints provided
+  by the v1 API downstream.  (It MAY assume no breaking downstream changes.)
```

Comment:
What do you think of:

```suggestion
  by the v1 API downstream.  (Any downstream breaking changes MUST be coordinated.)
```

Not that we have any planned, but I think if we did need to make a breaking change, the SDK wouldn't be what's standing in the way, if that makes sense.

Reply:
I think that's righter.  Thanks!